### PR TITLE
perf: optimize Envoy access log persistence performance

### LINKED ISSUE #7180: Improve LAL performance
We are using LAL to analyze log analysis in the ALS case, which faces a very serious performance impact in only 10k traffic. 
In a very simple case, sampling 5 logs in N seconds if logs show error status.

@kezhenxu94 I remember there is proto to YAML object process, I am not sure whether it causes a big impact. We need to check this later because we also have to resolve this in an OAL style rebuild solution. Also, because it seems we will break the LAL APIs totally, we are better to change in the 8.7.0 rather than in further releases.

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>. Closes #7180 
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/filter/FilterSpec.java [hunk 1]
```diff
@@ -161,9 +169,19 @@ public void sink(@DelegatesTo(SinkSpec.class) final Closure<?> cl) {
...
+        if (extraLog != null) {
+            logData.setBody(
+                LogDataBody.newBuilder()
+                           .setJson(
+                               JSONLog.newBuilder()
+                                      .setJson(toJSON(extraLog))
+                                      .build())
+                           .build()
+            );
+        }
```
Comment (wu-sheng) [inline oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/filter/FilterSpec.java:181]: Is this the last step before persistent?

Comment (kezhenxu94) [reply oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/filter/FilterSpec.java:181]: > Is this the last step before persistent?

Yes, it's last step of the LAL, maybe I can move it to right before persistence.

Comment (wu-sheng) [reply oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/filter/FilterSpec.java:181]: Could use have multiple LAL files? If so, let's move it out of LAL scope.

Comment (kezhenxu94) [reply oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/filter/FilterSpec.java:181]: DONE

File: oap-server/server-bootstrap/src/main/resources/lal/envoy-als.yaml [hunk 2]
```diff
@@ -17,24 +17,22 @@ rules:
   - name: envoy-als
     dsl: |
       filter {
-        json {
-        }
         // only collect abnormal logs (http status code >= 300, or commonProperties?.responseFlags is not empty)
-        if (parsed?.response?.responseCode as Integer < 400 && !parsed?.commonProperties?.responseFlags) {
```
Comment (wu-sheng) [inline oap-server/server-bootstrap/src/main/resources/lal/envoy-als.yaml:23]: Why `!parsed?.commonProperties?.responseFlags` removed?

Comment (wu-sheng) [reply oap-server/server-bootstrap/src/main/resources/lal/envoy-als.yaml:23]: Do you mean `< 400` means no internal error?

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/log/AbstractLogRecord.java [hunk 3]
```diff
@@ -44,6 +44,7 @@
     public static final String SPAN_ID = "span_id";
     public static final String CONTENT_TYPE = "content_type";
     public static final String CONTENT = "content";
+    public static final String CONTENT_TYPE_CLASS = "content_type_class";
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/log/AbstractLogRecord.java:47]: I think we don't need this? You run `#toJSON` already.

File: oap-server/server-bootstrap/src/main/resources/lal/envoy-als.yaml (hunk 2 above)
Comment (kezhenxu94) [reply oap-server/server-bootstrap/src/main/resources/lal/envoy-als.yaml:23]: Added back

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/log/AbstractLogRecord.java (hunk 3 above)
Comment (kezhenxu94) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/log/AbstractLogRecord.java:47]: Yes, forgot to remove, now removed

**Review by wu-sheng — APPROVED** [review]
LGTM. Wait for tests.

**Pull request merged**